build/ps-entrypoint.sh

```diff
@@ -180,9 +180,17 @@ create_default_cnf() {
 }
 
 load_group_replication_plugin() {
+	POD_IP=$(hostname -I | awk '{print $1}')
+
 	sed -i "/\[mysqld\]/a plugin_load_add=group_replication.so" $CFG
 	sed -i "/\[mysqld\]/a group_replication_bootstrap_group=OFF" $CFG
 	sed -i "/\[mysqld\]/a group_replication_start_on_boot=OFF" $CFG
+	sed -i "/\[mysqld\]/a group_replication_consistency=EVENTUAL" $CFG
+	sed -i "/\[mysqld\]/a group_replication_exit_state_action=READ_ONLY" $CFG
+	sed -i "/\[mysqld\]/a group_replication_communication_stack=XCOM" $CFG
+	sed -i "/\[mysqld\]/a group_replication_ip_allowlist=${POD_IP}/16" $CFG
+	sed -i "/\[mysqld\]/a group_replication_ssl_mode=REQUIRED" $CFG
+	sed -i "/\[mysqld\]/a group_replication_recovery_use_ssl=ON" $CFG
```

Comment:
@Tusamarco what do you think about these defaults?

Reply:
I do not like EVENTUAL however is the only safe setting for DDL https://www.percona.com/blog/online-ddl-with-group-replication-in-mysql-8-0-27/
xcom we are covering it elsewhere